File: QA-hibernate/qa-hibernate/src/main/java/com/example/qahibernate/entities/Answers.java
package com.example.qahibernate.entities;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

/**
 * Answers
 */
@Entity(name="answers")
public class Answers {

    @Id
    @GeneratedValue(strategy=GenerationType.IDENTITY)
    private Long id;

    @Column(name = "text")
    private String text;
    
    @Column(name = "question_id")
    private long questionId;

    @Column(name = "user_id")
    private long userId;
    

    public Long getId() {
        return this.id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getText() {
        return this.text;
    }

    public void setText(String text) {
        this.text = text;
    }

    public long getQuestionId() {
        return this.questionId;
    }

    public void setQuestionId(long questionId) {
        this.questionId = questionId;
    }

    public long getUserId() {
        return this.userId;
    }

    public void setUserId(long userId) {
        this.userId = userId;
    }



    
}

File: QA-typeORM/QuestionAnswer/src/entity/Answers.ts
import {Entity, PrimaryGeneratedColumn, Column, ManyToOne, JoinColumn, OneToMany} from "typeorm";
import { Questions } from "./Questions";
import { Users } from "./Users";


@Entity()
export class Answers {

    @PrimaryGeneratedColumn()
    id: number;

    @Column()
    text: string;

    @ManyToOne(type => Questions, questions => questions.answers)
    @JoinColumn({name: "question_id"})
    questions: Questions;

    @ManyToOne(type => Users, users => users.answers)
    @JoinColumn({name: "user_id"})
    users: Users[]



}